functions/src/send-email.ts

```diff
@@ -1,10 +1,12 @@
 import * as mailgun from 'mailgun-js'
-import * as functions from 'firebase-functions'
 
 const DOMAIN = 'mg.manifold.markets'
-const mg = mailgun({ apiKey: functions.config().mailgun.key, domain: DOMAIN })
 
 export const sendTextEmail = (to: string, subject: string, text: string) => {
+  const mg = mailgun({
```

Comment:
Would it be better to move this to a function, e.g. `initMailgun()` and then reuse below?

Reply:
OK, I buy that.